delta-app/src/main/java/io/cdap/delta/app/EventMetrics.java

```diff
@@ -52,12 +53,17 @@ public synchronized void incrementDDLCount() {
     ddlEventCount++;
   }
 
+  public synchronized void incrementDMLErrorCount() {
+    dmlErrorCount++;
+  }
+
   public synchronized void emitMetrics() {
     for (DMLOperation.Type op : dmlEventCounts.keySet()) {
-      metrics.count(String.format("dml.%s", op.name().toLowerCase()), dmlEventCounts.get(op));
+      metrics.count(String.format("dml.%ss", op.name().toLowerCase()), dmlEventCounts.get(op));
```

Comment:
does change of metrics name have any impact on UI side ?

Reply:
yes. Have talked to @elfenheart about it. Those changes will be done once the PR is merged.